src/index.ts

```diff
@@ -167,6 +163,17 @@ class Formsy extends React.Component<FormsyProps, FormsyState> {
       this.prevInputNames = newInputNames;
       this.validateForm();
     }
+
+    if (disabled !== prevProps.disabled) {
```

Comment:
So this is a bug-fix for when the form is disabled after mount?

Reply:
Not really. Before React called the function getChildContext() (where you can access the props) to create the context value. 
With the new Context API, the context value are in the state of our component. And we need to keep the disabled value of the state the same as the disabled value from the form props.
We cannot do that in the render method because it will create a new context value and re-render every Consumer.

Reply:
```suggestion
    // Keep the disabled value in state/context the same as from props
    if (disabled !== prevProps.disabled) {
```